mlflow/telemetry/schemas.py

```diff
@@ -66,3 +66,16 @@ class TelemetryInfo:
 class TelemetryConfig:
     ingestion_url: str
     disable_events: set[str]
+    batch_time_interval_seconds: int
+    retryable_error_codes: set[int]
+    stop_on_error_codes: set[int]
+
+    @classmethod
+    def from_dict(cls, config: dict[str, Any]) -> "TelemetryConfig":
+        return TelemetryConfig(
+            ingestion_url=config["ingestion_url"],
+            disable_events=set(config.get("disable_events", [])),
+            batch_time_interval_seconds=int(config["batch_time_interval_seconds"]),
```

Comment:
Is `batch_time_interval_seconds` not int?

Reply:
It is, just to be safe here

Reply:
Got it, let's add a comment.